tests/TestResponse.php

```diff
@@ -34,7 +34,7 @@ public function __construct(Application $app, Response $response)
         $this->app          = $app;
     }
 
-    public function assertSuccessful()
+    public function assertSuccessful(): self
```

Comment:
Not entirely sure about these assertions in the first place, personally I would move the assertions to a trait and use the trait in tests instead, rather than decorating the response object.